README.md

```diff
@@ -150,12 +150,38 @@ use on npm.
 | [**bemuse-indexer**](packages/bemuse-indexer)     | [![npm](https://img.shields.io/npm/v/bemuse-indexer.svg)](http://npmjs.com/package/bemuse-indexer)     | Library for scanning a directory for BMS/bmson files and generate a metadata.                   |
 | [**bemuse-notechart**](packages/bemuse-notechart) | [![npm](https://img.shields.io/npm/v/bemuse-notechart.svg)](http://npmjs.com/package/bemuse-notechart) | Library for loading a notechart for use in game.                                                |
 
-## License
+## How to contribute:
 
-[AGPLv3](LICENSE). Note that this license only applies to the main Bemuse
-project, not the [sub-projects](#sub-projects), which has its own license terms
-(mostly MIT).
+- Drop a :star: on the Github repository (optional)<br/>
+
+- Before Contribute Please read [CONTRIBUTING.md](https://github.com/bemusic/bemuse/blob/master/CONTRIBUTING.md) and [CODE_OF_CONDUCT.md](https://github.com/bemusic/bemuse/blob/master/CODE_OF_CONDUCT.md)
+
+- Create an issue of the project or a feature you would like to add in the project and get the task assigned for youself.(Issue can be any bug fixes or any feature you want to add in this project).
+
+- Fork the repo to your Github.<br/>
+
+- Clone the Repo by going to your local Git Client in a particular local folder in your local machine by using this command with your forked repository link in place of below given link: <br/>
+  `git clone https://github.com/<enter your git username here>/bemuse`
+- Create a branch using below command.
+  `git branch <your branch name>`
+- Checkout to your branch.
+  `git checkout <your branch name>`
+- Add your code in your local machine folder.
+  `git add . `
+- Commit your changes.
+  `git commit -m"<add your message here>"`
+- Push your changes.
+  `git push --set-upstream origin <your branch name>`
 
+- Make a pull request! (compare your branch with the owner main branch)
```

Comment:
While I appreciate the extra guidance (e.g. drop a star), we already have a developer guide on the website.

https://bemuse.ninja/project/docs/developer-guide.html

Let’s just link there instead. Feel free to make changes to the developers guide to make it easier for newcomers to work on the project.